utility/src/main/java/org/oppia/android/util/parser/image/BUILD.bazel

```diff
@@ -0,0 +1,225 @@
+        ":glide_image_loader",
+        ":image_loader",
+        ":text_picture_drawable",
+        "//third_party:javax_inject_javax_inject",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:block_picture_drawable",
+    ],
+)
+
+kt_android_library(
+    name = "image_parsing_annonations",
+    srcs = [
+        "ImageParsingAnnotations.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+    deps = [
+        "//third_party:javax_inject_javax_inject",
+    ],
+)
+
+kt_android_library(
+    name = "image_parsing_module",
+    srcs = [
+        "ImageParsingModule.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+    deps = [
+        ":dagger",
+        ":image_parsing_annonations",
+        "//third_party:javax_inject_javax_inject",
+    ],
+)
+
+kt_android_library(
+    name = "image_targets",
+    srcs = [
+        "ImageTargets.kt",
+    ],
+    visibility = ["//:oppia_testing_visibility"],
+    deps = [
+        ":image_asset_fetcher",
+        "//third_party:com_github_bumptech_glide_glide",
+    ],
+)
+
+kt_android_library(
+    name = "image_transformation",
+    srcs = [
+        "ImageTransformation.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+)
+
+kt_android_library(
+    name = "repository_glide_module",
+    srcs = [
+        "RepositoryGlideModule.kt",
+    ],
+    deps = [
+        ":image_asset_fetcher",
+        ":repository_model_loader",
+        "//third_party:glide_compiler",
+        "//utility/src/main/java/org/oppia/android/util/caching:annotations",
+        "//utility/src/main/java/org/oppia/android/util/caching:asset_repository",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:block_picture_drawable",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:block_svg_drawable_transcoder",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:scalable_vector_graphic",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:svg_decoder",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:text_svg_drawable_transcoder",
+    ],
+)
+
+kt_android_library(
+    name = "repository_model_loader",
+    srcs = [
+        "RepositoryModelLoader.kt",
+    ],
+    deps = [
+        ":image_asset_fetcher",
+        "//third_party:com_github_bumptech_glide_glide",
+    ],
+)
+
+kt_android_library(
+    name = "text_picture_drawable",
+    srcs = [
+        "TextPictureDrawable.kt",
+    ],
+    visibility = ["//utility/src/main/java/org/oppia/android/util/parser/svg:__pkg__"],
+    deps = [
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:scalable_vector_graphic",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:svg_picture_drawable",
+    ],
+)
+
+kt_android_library(
+    name = "image_loader",
+    srcs = [
+        "ImageLoader.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
```

Comment:
Shouldn't this be API visibility since it's used in places to load images?

Reply:
fixed it.